lib/dir_scanner.js

```diff
@@ -15,7 +15,7 @@ function readFile(filename, vars) {
     const items = JSON.parse(newContent);
     return items;
   } catch (error) {
-    return {};
+    throw new Error(`couldn't parse package.json at '${filename}': ${parseError.message}`);
```

Comment:
You copy-pasted an old version of my suggestion from the issue (i updated my comment since then). The config file is probably not a `package.json`, and the error-object isn't called `parseError` here. This would be more correct:

```JavaScript
throw new Error(`couldn't parse JSON-file at '${filename}': ${error.message}`);
```